shared/src/sources/balancer/pool_cache.rs

```diff
@@ -201,7 +202,10 @@ fn handle_results(results: Vec<FetchedBalancerPool>) -> Result<Vec<BalancerPool>
                             .expect("Stable pools must have this set."),
                     )? {
                         // This is the ratio of amplification_parameter / precision.
-                        Some(state) => state.0.to_big_rational() / state.2.to_big_rational(),
+                        Some((amplification_factor, _, precision)) => BigRational::new(
```

Comment:
Maybe this is something else that we can address in the refactor and allow `get_stable_pool_data` to be reused here, instead of duplicating logic for computing the amplification parameter.

Reply:
Yes, I was actually a bit confused as to why there were two places that need to be changed here. Will add it to the list.